SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -113,10 +115,10 @@
   <!-- **************************************************************************** -->
   <Target Name="WriteSonarQubeProjectData" AfterTargets="Build"
         Condition=" $(SonarQubeTempPath) != '' ">
-    
```

Comment:
The rest of the changes in this file are whitespace changes caused by converting tabs to spaces.